imagetool/src/main/java/com/oracle/weblogic/imagetool/cli/menu/CreateImage.java

```diff
@@ -326,9 +314,9 @@ public CommandResponse call() throws Exception {
      * @param wdtKey key in the format wdt_0.17
      * @throws Exception in case of error
      */
-    private void addWDTURL(String wdtKey) throws Exception {
+    private void addWdtUrl(String wdtKey) throws Exception {
```

Comment:
I am wondering why the wdt url is added to the cache store but never used.

Reply:
good question, but I will file an issue to address this question rather than dealing with it here.